Azure/azure-cli-extensions
# [Logz.io] Release an experimental version

> Closed https://github.com/Azure/azure-cli/issues/19266

**Description**

Draft pull request for Logz.io.

**Testing Guide**

```
$ azdev test logz
```
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines


For new extensions:



### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.
